src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -101,7 +101,7 @@ public ResourceElement CreateHistoryBundle(IEnumerable<Tuple<string, string>> un
                         Request = new Bundle.RequestComponent
                         {
                             Method = hasVerb ? (Bundle.HTTPVerb?)httpVerb : null,
-                            Url = r.Request?.Url?.ToString(),
+                            Url = hasVerb ? $"{resource.InstanceType}/{(httpVerb == Bundle.HTTPVerb.POST ? null : resource.Id)}" : null,
```

Comment:
what's this change for?

Reply:
We should be returning return relative paths here (and we do not store it in SQL)